### DISCUSSION TITLE
How to access the postgres and clickhouse database in self hosted  version

### DISCUSSION THREAD

**Discussion by Elucidator2197** [plausible/analytics#621] [Self-Hosted Support] [answered]
Want to see the data stored in the databases but unable to find it on amazon ec2 instance with unix platform

**Comment by ukutaht** [comment, marked as answer]
### Postgres

Run `$ docker ps` to figure out the container name for the Postgres image. In my case it was `hosting_plausible_db_1`. Then you can connect as follows

```sh
$ docker exec -ti hosting_plausible_db_1 psql -U postgres plausible_db
```

 ### Clickhouse
Run `$ docker ps` to figure out the container name for the Clickhouse image. In my case it was `hosting_plausible_db_1`.

Also figure out the name of your docker-compose network with `$ docker network ls`. In my case it was `hosting_default` Then you can connect as follows

```sh
$ docker pull yandex/clickhouse-client
$ docker run -it --net hosting_default --rm --link hosting_plausible_events_db_1:clickhouse-server yandex/clickhouse-client --host clickhouse-server --database plausible_events_db

```

**Reply by kees-closed** [reply]
Is there also an official documentation page that shows how to do daily backups and a restore?

**Reply by ruslandoga** [reply]
@kees-closed https://clickhouse.com/docs/en/operations/backup/ might be useful.
